packages/devtools/src/ObservableHQ.tsx

```diff
@@ -3,22 +3,34 @@ import { ctx, FC, h, mount, JSX, css, ROOT } from './jsx'
 
 // @ts-expect-error TODO write types
 import { Inspector } from '@observablehq/inspector'
-// @ts-expect-error
-import observablehqStyles from '../../../node_modules/@observablehq/inspector/dist/inspector.css'
+import observablehqStyles from '../../../node_modules/@observablehq/inspector/dist/inspector.css?raw'
```

Comment:
It was a hack for Bun
```suggestion
import observablehqStyles from '@observablehq/inspector/dist/inspector.css?raw'
```